container-tools/debian-confd/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# ConfD Daemon Container Image
+
+# Start from the debian stretch linux base image.  (Unfortunately we can't go
+        # beyond stretch until ConfD support libcrypto 1.1.)
+FROM debian:9
```

Comment:
Use Debian 10 "Buster" instead, as ConfD 7.2 support libcrypto 1.1 if you rebuild crypto